File: C# ad y ad/2_ Orientado a objetos/S05.1_ La observación/Solucion_la_observacion/La_observacion/Src/Arbol/Arbol.cs

namespace La_observacion
{
	class Arbol
	{
		float altura;
		string cantidadDeHojas;


		public Arbol(float altura, string cantidadDeHojas)
		{
			this.altura = altura;
			this.cantidadDeHojas = cantidadDeHojas;
		}


		public float GetAltura()
		{
			return altura;
		}
		public string GetCantidadDeHojas()
		{
			return cantidadDeHojas;
		}
	}
}

File: C++ ad y ad/3_ Orientado a objetos/S05.1_ La observacion/Solucion_la_observacion/La_observacion/src/arbol/arbol.cpp
#include "arbol.h"

#include <iostream>


Arbol::Arbol(float altura, string cantidadDeHojas)
{
	this->altura = altura;
	this->cantidadDeHojas = cantidadDeHojas;
}
Arbol::~Arbol()
{
	cout << "El arbol con " << altura << " m. de altura y " << cantidadDeHojas << " hojas fue liberado de la memoria." << endl << endl;
}


float Arbol::getAltura()
{
	return altura;
}
string Arbol::getCantidadDeHojas()
{
	return cantidadDeHojas;
}
